lib/artist.rb

```diff
@@ -0,0 +1,49 @@
+require 'pry'
+
+class Artist
+  extend Concerns::Findable
+  # wanted to make `Song` the source of truth, but tests
+  # are invoking `<<` on `@songs`, which I can't find a better way arround...
```

Comment:
Yeah - Flatiron labs break the source of truth concept, even though they also talk about it.